data/download.yaml

```diff
@@ -0,0 +1,42 @@
+
+binaries:
+  platforms:
+  - darwin
+  - linux
+  - windows
+  platformInfo:
+    darwin:
+      name: macOS
+      icon: fa-apple
+    linux:
+      name: Linux
+      icon: fa-linux
+    windows:
+      name: Windows
+      icon: fa-windows
+  fullReleaseVersions:
+  - "1.4.1"
+  - "1.5.0"
+  - "1.6.0"
+  partialReleaseVersions:
```

Comment:
these ones have almost no value since the archives are just the source code, we only started packaging executables in from 1.4.1

Reply:
Okay, I can remove the "partial release" versions. I assumed you may not want them but figured I'd err on the side of including first and potentially removing later.